content/docs/pulumi-cloud/reference/deployments/_index.md

```diff
@@ -0,0 +1,1043 @@
+
+The operation context describes any context required for Pulumi operations to execute such as pre-run commands and environment variables.
+
+```json
+{
+  "preRunCommands": [
+    "echo \"hello world\""
+  ],
+  "environmentVariables": {
+    "AWS_REGION": "us-west-2",
+    "MY_PASSWORD": {
+      "secret": "my-secret-password"
+    }
+  },
+  "options": {
+    "skipInstallDependencies": false,
+    "skipIntermediateDeployments": false,
+    "deleteAfterDestroy": false
+  },
+  "oidc": {
+    "aws": {
+      "roleArn": "arn:aws:iam::123456789000:role/pulumi-deploy-role",
+      "sessionName": "pulumi-deploy-session"
+    }
+  }
+}
+```
+
+#### Properties
+
+| Name               | Type           | Description                                      |
+|--------------------|----------------|--------------------------------------------------|
+| `preRunCommands`   | string[] | **Optional.** Commands to run before Pulumi execution. |
+| `environmentVariables` | object | **Optional.** Environment variables for the operation. |
+| `options` | object | **Optional.** Operation context options. |
+| `options.skipInstallDependencies` | boolean | **Optional.** Skip automated dependency installation. |
+| `options.skipIntermediateDeployments` | boolean | **Optional.** Skip intermediate deployments. |
+| `options.deleteAfterDestroy` | boolean | **Optional.** Delete stack after destroy operation. |
+| `oidc` | object | **Optional.** OIDC configuration for cloud provider authentication. |
+
+### GitHub
+
+The GitHub block describes settings for Pulumi Deployments' GitHub integration.
+
+```json
+{
+  "repository": "pulumi/deploy-demos",
+  "deployCommits": true,
+  "previewPullRequests": true,
+  "pullRequestTemplate": false,
+  "paths": [ "pulumi-programs/bucket-time/**", "!pulumi/programs/bucket-time/README.md" ]
+}
+```
+
+#### Properties
+
+| Name                | Type           | Description                                      |
+|---------------------|----------------|--------------------------------------------------|
+| `repository`        | string | **Required.** The GitHub repository containing the Pulumi program. |
+| `deployCommits`     | boolean | **Optional.** Run update deployments for commits to the configured branch. |
+| `previewPullRequests` | boolean | **Optional.** Run preview deployments for PRs targeting the configured branch. |
+| `pullRequestTemplate` | boolean | **Optional.** Enable Review Stacks for this branch. |
+| `paths`             | string[] | **Optional.** Path filters for triggering deployments. |
+
+### CacheOptions
+
+The cache options block defines settings related to dependency caching during deployments.
+
+```json
+{
+  "cacheOptions": {
+    "enable": "true"
+  }
+}
+```
+
+#### Properties
+
+| Name      | Type    | Description                                      |
+|-----------|---------|--------------------------------------------------|
+| `enable`  | boolean | **Required.** Whether to use dependency caching. |
```

Comment:
This file is about 500 lines shorter than the original, and seems to be in a different ordering/format. Given the lack of ability to diff these changes, it might take some time to review this manually to ensure there was no loss of information during this conversion. I'm not a subject matter expert on this API. Should we ask an engineer from the team to review this?

Reply:
@komalali Can you review these changes and/or do a spot check?

@thoward Alternatively, we can agree that the risk of a potential error in the content (which can be corrected later) is worth the benefit of getting our REST API docs in one place.